src/common/checksums.cpp

```diff
@@ -142,20 +142,26 @@ QByteArray makeChecksumHeader(const QByteArray &checksumType, const QByteArray &
     return header;
 }
 
-QByteArray findBestChecksum(const QByteArray &checksums)
+QByteArray findBestChecksum(const QByteArray &_checksums)
 {
+    const auto checksums = QString::fromUtf8(_checksums);
```

Comment:
I find this `auto` confusing, because I have to think about to where fromUtf8 converts.